src/babel/api/register/node.js

```diff
@@ -160,7 +161,7 @@ hookExtensions(util.canCompile.EXTENSIONS);
 
 export default function (opts = {}) {
   if (opts.only != null) only = util.arrayify(opts.only, util.regexify);
-  if (opts.ignore != null) ignore = util.arrayify(opts.ignore, util.regexify);
+  if (opts.ignore != null) ignore = util.arrayify(opts.ignore, util.regexify, isFunction);
```

Comment:
Would it be easier to lump the `isFunction` check into the `mapFn`? Adding a third param seems unintuitive to me. e.g.

    util.arrayify(opts.ignore, item => isFunction(item) ? item : util.regexify(item));